### ISSUE TITLE
Game Objects / Tilemap / Dynamic / Draw Tile.js

### ISSUE THREAD

**Issue by davidwestAC** [photonstorm/phaser3-examples#100]
src\game objects\tilemap\dynamic\Draw Tile.js contains git merge conflict markers and won't load normally.

Example lines 83-87:
```javascript
<<<<<<< HEAD
    this.input.on('pointermove', function (pointer) {
=======
    this.input.on('MOUSE_MOVE_EVENT', function (event) {
>>>>>>> pr/11
```

**Comment by gabegordon** [comment]
The file that you referenced does not exist in that path.

**Comment by dazigemm** [comment]
It seems like the example was removed in 78fea24 to fix issue #28

**Comment by davidwestAC** [comment]
Yeah the example/file no longer exists. Thanks for the reason dazigemm.
